cardano-api/src/Cardano/Api/LedgerEvent.hs

```diff
@@ -153,23 +155,33 @@ pattern LEMirTransfer rp tp rtt ttr <-
         ( MirEvent
             ( MirTransfer
                 ( InstantaneousRewards
-                    (convertSumRewardsMap -> rp)
-                    (convertSumRewardsMap -> tp)
+                    (convertSumRewardsMapSimple -> rp)
+                    (convertSumRewardsMapSimple -> tp)
                     (fromShelleyDeltaLovelace -> rtt)
                     (fromShelleyDeltaLovelace -> ttr)
                   )
               )
           )
       )
 
-convertSumRewardsMap ::
+convertRewardEventMapNew ::
+  Map
+    ( Cardano.Ledger.Credential.StakeCredential
+        Cardano.Ledger.Crypto.StandardCrypto
+    )
+    (Set (Reward StandardCrypto)) ->
+  Map StakeCredential (Set (Reward StandardCrypto))
+convertRewardEventMapNew =
+  Map.mapKeys fromShelleyStakeCredential
+
+convertSumRewardsMapSimple ::
   Map
     ( Cardano.Ledger.Credential.StakeCredential
         Cardano.Ledger.Crypto.StandardCrypto
     )
     Cardano.Ledger.Coin.Coin ->
   Map StakeCredential Lovelace
-convertSumRewardsMap =
+convertSumRewardsMapSimple =
```

Comment:
Likewise with `Simple` here

Reply:
We need that `Simple` to differentiate it from `convertRewardEventMap` which is different.

Reply:
They have different names - `convertRewardEventMap` and `convertSumRewardsMap`